python/python/lance/vector.py

```diff
@@ -209,6 +213,8 @@ def compute_partitions(
     dst_dataset_uri: Union[str, Path], optional
         The path to store the partitions.  If not specified a random
         directory is used instead
+    num_workers: int, default 4
+        Number of workers for PyTorch DataLoader.
```

Comment:
there is no arg called `num_workers`

Reply:
Ok, this is left-over. i removed the multi-worker from loader, did not show noticeable speed up on my machine. Gonna try on a L4 machine.